src/web/client/common/errorHandler.ts

```diff
@@ -83,43 +83,23 @@ export function removeEncodingFromParameters(
 
 export function checkMandatoryParameters(
     appName: string,
-    entity: string,
-    entityId: string,
     queryParamsMap: Map<string, string>
 ): boolean {
     return (
-        checkMandatoryPathParameters(appName, entity, entityId) &&
+        checkMandatoryPathParameters(appName) &&
         checkMandatoryQueryParameters(appName, queryParamsMap) &&
```

Comment:
checkMandatoryQueryParameters needs to be enhanced as well -IF you have entityId, entitiyName and OrgURL, I do not think websiteId is the mandatory param - confirm this once

Reply:
as per current implementation we need websiteid as a mandatory parameter.

Reply:
as per discussion website id will help to narrow down the portal recoreds